router/core/src/main/scala/io/buoyant/router/Router.scala

```diff
@@ -286,7 +291,7 @@ object StackRouter {
 
     case class PathParams(prefix: PathMatcher, mk: Map[String, String] => Stack.Params)
 
-    case class PerPathParams(params: Seq[PathParams]) {
+    case class PerPathParams(params: Seq[PathParams], dynamicParams: Option[Activity[Seq[PathParams]]]) {
```

Comment:
Cool I will try out the idea. Originally I thought io.l5d.fs could have config defined in BOTH a file and also statically, but dropped that idea due to its complexity. So I bet separating them is no longer needed.